exercise1/romanNumberEncoder.js

```diff
@@ -0,0 +1,25 @@
+function romanNumberEncoder(number) {
+  const romanNumerals = {
+    M: 1000,
+    CM: 900,
+    D: 500,
+    CD: 400,
+    C: 100,
+    XC: 90,
+    L: 50,
+    XL: 40,
+    X: 10,
+    IX: 9,
+    V: 5,
+    IV: 4,
+    I: 1,
+  };
+  let output = "";
+  for (const roman in romanNumerals) {
+    while (number >= romanNumerals[roman]) {
+      output += roman;
+      number -= romanNumerals[roman];
+    }
+  }
```

Comment:
[OPTIONAL]  Do you mind giving this variable a meaningful name? Its a suggestion but in case you find it okay to use output then its fine.